<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HistoryListItem</class>
 <widget class="QWidget" name="HistoryListItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1102</width>
    <height>43</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border: 0px;
background: none;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="1,1" rowminimumheight="0,1" columnminimumwidth="0,0,0,0,0">
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <property name="horizontalSpacing">
    <number>4</number>
   </property>
   <property name="verticalSpacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="lblBrand">
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>10</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Brand</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignBottom|Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="lblModel">
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>10</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Model</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="lblPrice">
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>10</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Price</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="lblDateFrom">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>10</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Datum</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="lblDateTo">
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>10</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Datum</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTop|Qt::AlignmentFlag::AlignTrailing</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_4">
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>7</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Marke</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_5">
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>7</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Modell</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label_6">
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>7</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Preis</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="label_7">
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>7</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Ausleihdatum</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QLabel" name="label_8">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>7</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>RÃ¼ckgabedatum</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTop|Qt::AlignmentFlag::AlignTrailing</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
